holidays/countries/mongolia.py

```diff
@@ -0,0 +1,93 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import InternationalHolidays, MongolianCalendarHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Mongolia(HolidayBase, InternationalHolidays, MongolianCalendarHolidays):
+    """Mongolia holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Mongolia>
+        * <https://www.timeanddate.com/holidays/mongolia>
+        * <https://investmongolia.gov.mn/mongolia-at-a-glance/>
+        * <https://www.math.mcgill.ca/gantumur/cal/year.html>
+    """
+
+    country = "MN"
+    default_language = "mn"
+    start_year = 2004
+    supported_languages = ("en_MN", "en_US", "mn")
```

Comment:
I don't see the relevance of `en_MN`. We support local versions of English in countries where it is the primary or widely spoken language. Mongolia is clearly not one of them.